deploy/cdk/lib/manifest-pipeline/manifest-pipeline-stack.ts

```diff
@@ -0,0 +1,713 @@
+      code: Code.fromInline(`'use strict';
+        exports.handler = (event, context, callback) => {
+          var request = event.Records[0].cf.request;
+
+          if (!request.uri.endsWith('/index.json') && !request.uri.endsWith('.xml')) {
+            if (request.uri.endsWith('/')) {
+              request.uri = request.uri + 'index.json'
+            } else {
+              request.uri = request.uri + '/index.json'
+            }
+          }
+          return callback(null, request);
+        };`),
+      handler: 'index.handler',
+      runtime: Runtime.NODEJS_12_X,
+      description: `This Lambda will take incoming web requests and adjust the request URI as appropriate.
+        Any directory that does not end with an index.json will have that appended to it.`,
+    })
+
+    const sPARedirectionLambdaV2 = new Version(this, 'SPARedirectionLambdaV2', {
+      lambda: sPARedirectionLambda,
+      description: 'Adds rewrite rules as needed',
+    })
+
+
+    // Create a CloudFront Distribution for the manifest bucket
+    const fqdn = `${props.hostnamePrefix}.${props.domainName}`
+    const distribution = new CloudFrontWebDistribution(this, 'CloudFrontWebDistribution', {
+      originConfigs: [{
+        s3OriginSource: {
+          s3BucketSource: this.manifestBucket,
+          originAccessIdentity: originAccessId,
+        },
+        behaviors: [{
+          isDefaultBehavior: true,
+          lambdaFunctionAssociations: [{
+            eventType: LambdaEdgeEventType.ORIGIN_REQUEST,
+            lambdaFunction: sPARedirectionLambdaV2,
+          }],
+          forwardedValues: {
+            queryString: false,
+            headers: ["Access-Control-Request-Headers",
+              "Access-Control-Request-Method",
+              "Origin",
+              "Authorization",
+            ],
+          },
+          allowedMethods: CloudFrontAllowedMethods.GET_HEAD_OPTIONS,
+          compress: true,
+        }],
+      }],
+      defaultRootObject: 'index.json',
+      httpVersion: HttpVersion.HTTP2,
+      loggingConfig: {
+        bucket: props.foundationStack.logBucket,
+        includeCookies: true,
+        prefix: `web/${props.hostnamePrefix}.${props.domainName}`,
+      },
+      priceClass: PriceClass.PRICE_CLASS_100,
+      viewerCertificate: ViewerCertificate.fromAcmCertificate(props.foundationStack.certificate, {
+        aliases: [fqdn],
+      }),
+      comment: props.hostnamePrefix + '.' + props.domainName,
+    })
+
+
+    if (props.createDns) {
+      new CnameRecord(this, `HostnamePrefix-Route53CnameRecord`, {
+        recordName: props.hostnamePrefix,
+        domainName: distribution.distributionDomainName,
+        zone: ManifestPipelineStack.hostedZoneToUse,
+        ttl: Duration.minutes(15),
+      })
+    }
+    
+    const initManifestLambda = new Function(this, 'InitManifestLambdaFunction', {
+      code: Code.fromAsset('../../../marble-manifest-pipeline/init/'),
```

Comment:
I think this should be using `props.lambdaCodeRootPath`

Reply:
Good catch.  I missed this one.  Thanks.

Reply:
I made these recommended changes.  Thanks.